src/fontra/views/editor/editor.html

```diff
@@ -47,16 +48,36 @@
                 <!-- content will be filled in dynamically -->
               </div>
               <div id="zoom-tools" class="tools-item">
-                <div class="tool-button" data-tool="zoom-out">
+                <div
+                  class="tool-button"
+                  data-tool="zoom-out"
+                  data-tooltip="Zoom out"
+                  data-tooltipposition="bottom"
+                >
                   <inline-svg class="tool-icon" src="/images/minus.svg"></inline-svg>
                 </div>
-                <div class="tool-button" data-tool="zoom-in">
+                <div
+                  class="tool-button"
+                  data-tool="zoom-in"
+                  data-tooltip="Zoom in"
+                  data-tooltipposition="bottom"
+                >
                   <inline-svg class="tool-icon" src="/images/plus.svg"></inline-svg>
                 </div>
-                <div class="tool-button" data-tool="zoom-fit-selection">
+                <div
+                  class="tool-button"
+                  data-tool="zoom-fit-selection"
+                  data-tooltip="Zoom fit selection"
```

Comment:
I wonder if a better title would bo "Zoom to fit", or even "Zoom to fit selection"?

Reply:
Sure, will change this.